Entities/Common/Scripts/AlertAdminOnDie.as

```diff
@@ -0,0 +1,58 @@
+const SColor CHAT_COLOR = SColor(255, 255, 0, 0);
+
+void onAttach(CBlob@ this, CBlob@ attached, AttachmentPoint@ attachedPoint)
+{
+	AttributeBlobToPlayer(this, attached.getPlayer());
+}
+
+void onThisAddToInventory(CBlob@ this, CBlob@ inventoryBlob)
+{
+	AttributeBlobToPlayer(this, inventoryBlob.getPlayer());
+}
+
+void onDie(CBlob@ this)
+{
+	string message;
+
+	string username = this.get_string("last held by");
+	CPlayer@ player = getPlayerByUsername(username);
+	if (player !is null)
+	{
+		message = getTranslatedString("{PLAYER} caused {COUNT} {MATERIAL} to fall into the void!")
+			.replace("{PLAYER}", formatPlayerName(player))
+			.replace("{COUNT}", "" + this.getQuantity())
+			.replace("{MATERIAL}", this.getInventoryName());
+	}
+	else
+	{
+		message = getTranslatedString("{COUNT} {MATERIAL} fell into the void!")
+			.replace("{COUNT}", "" + this.getQuantity())
+			.replace("{MATERIAL}", this.getInventoryName());
+	}
+
+	if (isServer())
+	{
+		print(message, CHAT_COLOR);
+	}
+
+	@player = getLocalPlayer();
+	if (player !is null && player.isMod())
+	{
+		client_AddToChat(message, CHAT_COLOR);
+	}
```

Comment:
I don't see the point of reusing the same `CPlayer@`, it just adds to confusion...
```suggestion
	CPlayer@ localPlayer = getLocalPlayer();
	if (localPlayer !is null && localPlayer.isMod())
	{
		client_AddToChat(message, CHAT_COLOR);
	}
```